packages/appChrome/components/SidebarSubMenuItem.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import {
+  subMenuItem,
+  subMenuItemText,
+  appChromeInsetContent,
+  sidebarNavItem
+} from "../style";
+import { cx } from "emotion";
+import { greyLightDarken2 } from "../../shared/styles/design-tokens-dist/js/designTokens";
+import { tintText, textSize, flex } from "../../shared/styles/styleUtils";
+import Clickable from "../../clickable/components/clickable";
+
+export interface SidebarSubMenuItemProps {
+  children?: React.ReactElement<HTMLElement> | string;
+  isActive?: boolean;
+  onClick: (event?: React.SyntheticEvent<HTMLElement>) => void;
+}
+
+class SidebarSubMenuItem extends React.PureComponent<
+  SidebarSubMenuItemProps,
+  {}
+> {
+  public render() {
+    const { children, isActive, onClick } = this.props;
+    const classNames = cx(
+      subMenuItem,
+      appChromeInsetContent,
+      sidebarNavItem(isActive),
+      // TODO: ask Lily about using our secondary text color/s instead
```

Comment:
Is fixing this in the scope of this ticket or just a reminder for the future?

Reply:
Lily and I have a meeting to review some of these design-related "TODO"s at 10:30(PST)

Reply:
After the meeting will you update this PR or make a new one?

Reply:
Update this one

Reply:
So it's still work in progress.

Reply:
Yes, but the only things left to do should be minor visual design tweaks. If there are any changes that take more than a few minutes to add, they'll go into a new ticket

Reply:
Okay, cool. Still, adding new things dismisses old reviews which is why I asked.